data/processed/aps/aps_econ_active_stem_density_data.yaml

```diff
@@ -0,0 +1,26 @@
+api_doc_url: https://www.nomisweb.co.uk/api/v01/help
+api_type: fetch
+description_short: STEM employee density
+description: Percentage of population employed in science, research, engineering and technology professional by NUTS 2 regions.
+endpoint_url: https://www.nomisweb.co.uk/api/v01/dataset/NM_17_5.jsonstat.json?geography=1837105153...1837105192&date=latestMINUS23,latestMINUS19,latestMINUS15,latestMINUS11,latestMINUS7,latestMINUS3&variable=1543&measures=20599,21001,21002,21003
+framework_group: 2
+is_experimental: False
+order: [year, nuts_id, nuts_year_spec, value.id] # do not change; specifies the order of the exported indicator fields
+region:
+  type: NutsRegion
+  level: 2

```

Comment:
we need to provide `source_url` or `source` here

Reply:
Fixed- made change in all schemas (`source` = PROVIDED).